app/views/events/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="container">
+  <h1> <%= @event.name %> </h1>
+  <h4>Event Creator: <%= link_to @event.creator.name, @event.creator %></h4>
+  <table class="table table-bordered">
+  <thead>
+    <tr>
+      <th> Description </th>
+      <th> Location </th>
+      <th> When </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><%= @event.description %> </td>
+      <td><%= @event.location %></td>
+       <td><%= @event.starts_at %> </td>
+    </tr>
+  </tbody>
+</table>
+
+<p><strong>Attendees: </strong></p>
+
+<ul>
+  <% @event.attendees.each do |attendee| %>
+		<li>
+			<%= link_to attendee.name, attendee %>
+		</li>
+  <% end %>
+</ul>
+
+
+
+<% if !event_creator? %>
+  <% if @current_attendee %>
+    <p> You have registered for this event </p>
+    <% else %>
+      <%= link_to 'Register for Event', event_join_path({event_id: @event.id, user_id: current_user.id}), class: 'btn btn-secondary'%>
+    <% end %>
+    
+<% end %>
```

Comment:
- [ ] create a helper method that has the if/else logic and render the links and markup within the helper method instead of doing it straight in the view